test/integration_test.py

```diff
@@ -1565,12 +1570,57 @@ def assert_manifest(expected_bundles):
         # With authorized credentials, all bundles included in the filters
         # should be represented in the manifest
         with self._service_account_credentials:
-            assert_manifest({public_bundle, *managed_access_bundles})
+            test_compact_manifest(all_bundles)
 
         # Without credentials, only the public bundle should be represented
-        assert_manifest({public_bundle})
+        test_compact_manifest({public_bundle})
+
+        def get_verbatim_manifest(bundles: Iterable[str],
+                                  format: ManifestFormat
+                                  ) -> BytesIO:
+            manifest_url = furl(url=endpoint, path='/manifest/files', args={
+                'catalog': catalog,
+                'format': format.value,
+                'filters': json.dumps({special_fields.bundle_uuid: {'is': list(bundles)}})
+            })
+            return BytesIO(self._get_url_content(PUT, manifest_url))
 
-        if ManifestFormat.curl in self.metadata_plugin(catalog).manifest_formats:
+        def read_verbatim_jsonl_manifest(manifest: IO) -> set[JSON]:
+            manifest_lines = manifest.readlines()
+            manifest_content = {
+                freeze(json.loads(replica))
+                for replica in manifest_lines
+            }
+            self.assertEqual(len(manifest_lines), len(manifest_content))
+            return manifest_content
+
+        def read_verbatim_pfb_manifest(manifest: IO) -> set[str]:
+            entities = list(fastavro.reader(manifest))
+            manifest_content = {
+                # We can't assert the full contents of each entity because the
+                # schema changes depending on the filters used.
+                entity['id']
+                for entity in entities
+                # The special "Metadata" entity is always present. Dropping it
+                # from the result streamlines the set logic used in the
+                # assertion below.
+                if entity['name'] != 'Metadata'
+            }
+            return manifest_content
+
+        for format, reader in [
+            (ManifestFormat.verbatim_jsonl, read_verbatim_jsonl_manifest),
+            (ManifestFormat.verbatim_pfb, read_verbatim_pfb_manifest)
+        ]:
+            if format in metadata_plugin.manifest_formats:
+                with self.subTest(format=format):
+                    unauthorized = reader(get_verbatim_manifest(all_bundles, format))
+                    with self._service_account_credentials:
+                        authorized = reader(get_verbatim_manifest(all_bundles, format))
+                        private_only = reader(get_verbatim_manifest(managed_access_bundles, format))
```

Comment:
I think we should consider filtering by `accessible` in addition to filtering by bundles, and compare the results. They should be identical.

I don't know if the manifest endpoint supports filtering by `accessible`. If it does, we should add this now.

Reply:
It does support the filter, but because the manifest is file-based, filtering by `accessible=[true]`, `accessible=[true, false]`, and not filtering by accessible will all give the same result, and filtering by `accessible=False`  will always return an empty manifest. This severely limits what we're able to assert using this filter.

Reply:
Agreed at PL that @hannes-ucsc will consider this further during his next review.